i3pystatus/core/__init__.py

```diff
@@ -106,22 +106,32 @@ def register(self, module, *args, **kwargs):
         if not module:
             return
 
+        err_colour = "#FF0000"
         try:
             return self.modules.append(module, *args, **kwargs)
         except ImportError as import_error:
             if import_error.name and not import_error.path and isinstance(module, str):
                 # This is a package/module not found exception raised by importing a module on-the-fly
                 return self.modules.append(Text(
-                    color="#FF0000",
+                    color=err_colour,
                     text="{i3py_mod}: Missing Python module '{missing_module}'".format(
                         i3py_mod=module,
                         missing_module=import_error.name)))
             else:
                 raise import_error
```

Comment:
I wrote it to only handle import errors that happened if you specified a module as a string. if you specify an instance you'd already have imported it and it would already have crashed before i3pystatus got involved.

Not sure why I wrote it to *only* handle this kind of error and not all ImportErrors.